tmol/score/ref/ref_energy_term.py

```diff
@@ -0,0 +1,105 @@
+import torch
+import numpy
+
+from ..energy_term import EnergyTerm
+
+from tmol.database import ParameterDatabase
+from tmol.score.ref.ref_whole_pose_module import RefWholePoseScoringModule
+
+from tmol.chemical.restypes import RefinedResidueType
+from tmol.pose.packed_block_types import PackedBlockTypes
+from tmol.pose.pose_stack import PoseStack
+
+from collections import OrderedDict
+
+
+class RefEnergyTerm(EnergyTerm):
+    device: torch.device  # = attr.ib()
+
+    def __init__(self, param_db: ParameterDatabase, device: torch.device):
+        super(RefEnergyTerm, self).__init__(param_db=param_db, device=device)
+
+        """self.global_params = RefGlobalParams.from_database(
+            param_db.scoring.ref, device
+        )"""
+        self.device = device
+
+        self.ref_weights = OrderedDict()
+
+        # 1.32468 3.25479 -2.14574 -2.72453 1.21829 0.79816 -0.30065 2.30374 -0.71458 1.66147 1.65735 -1.34026 -1.64321 -1.45095 -0.09474 -0.28969 1.15175 2.64269 2.26099 0.58223
+
+        self.ref_weights["ALA"] = 1.32468
+        self.ref_weights["ARG"] = 3.25479
+        self.ref_weights["ASN"] = -2.14574
+        self.ref_weights["ASP"] = -2.72453
+        self.ref_weights["CYS"] = 1.21829
+        self.ref_weights["GLN"] = 0.79816
+        self.ref_weights["GLU"] = -0.30065
+        self.ref_weights["GLY"] = 2.30374
+        self.ref_weights["HIS"] = -0.71458
+        self.ref_weights["ILE"] = 1.66147
+        self.ref_weights["LEU"] = 1.65735
+        self.ref_weights["LYS"] = -1.34026
+        self.ref_weights["MET"] = -1.64321
+        self.ref_weights["PHE"] = -1.45095
+        self.ref_weights["PRO"] = -0.09474
+        self.ref_weights["SER"] = -0.28969
+        self.ref_weights["THR"] = 1.15175
+        self.ref_weights["TRP"] = 2.64269
+        self.ref_weights["TYR"] = 2.26099
+        self.ref_weights["VAL"] = 0.58223
```

Comment:
Obviously temporary.

Reply:
If anyone has a pointer of where to find the values in Rosetta3 I can work on the actual DB implementation.